packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -93,31 +94,31 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
    * Store precomputed latency value
    * @param key - key
    * @param value - value
-   * @param overwrite - overwrite existing value or add it
+   * @param accumulate - when it true, it overwrite existing value with sum of the current value and the new measurement otherwise just store the new measurement
    * @throws
    * @returns
    */
-  public saveLatency(key: PreComputedLatencies, value: number, overwrite = true) {
-    const existingValue = overwrite ? 0 : this.precomputedLatencies.get(key) || 0;
+  public saveLatency(key: PreComputedLatencies, value: number, accumulate = true) {
```

Comment:
I've just realised, we've changed the meaning here too.
Should I accumulate? 
  - accumulate = true it overrides.
  - accumulate = false, it accumulates.
  
Also the default for this method should be to overwrite.

Reply:
So maybe change the default value and swap the tertiary statement outcomes on line 102.

Reply:
good point, I fixed all of them